engines/director/detection_tables.h

```diff
@@ -3236,6 +3284,8 @@ static const DirectorGameDescription gameDescriptions[] = {
 
 	WINGAME1_l("bigmax", "nº20", "BIG32.EXE", "518a98696fe1122e08410b0f157f21bf", 7313980, Common::PT_BRA, 602),
```

Comment:
You have non-ascii symbol here in the second parameter. If you have to have it, then please replace it with the octal codes